src/main/java/io/fabric8/vertx/maven/plugin/mojos/AbstractRunMojo.java

```diff
@@ -423,14 +422,22 @@ private boolean isYaml(String configFile) {
 
         @Override
         public void call(String kind, Path path) {
```

Comment:
Original intent was to get the file path to do some manipulations, since we moved that logic to builder this quiet good change to make.